ethereum/spec/src/main/java/tech/pegasys/teku/spec/config/SpecConfigFuluImpl.java

```diff
@@ -145,6 +148,11 @@ public SpecMilestone getMilestone() {
     return SpecMilestone.FULU;
   }
 
+  @Override
+  public UInt64 getBalancePerAdditionalCustodyGroup() {
+    return balancePerAdditionalCustodyGroup;
+  }
+
```

Comment:
`validatorCustodyRequirement`,
        `samplesPerSlot`,
        `balancePerAdditionalCustodyGroup`

        are missed from `equals` and `hashCode`
       we forgot about it before but now I've noticed